pkg/mflag/flag.go

```diff
@@ -503,6 +504,10 @@ func Set(name, value string) error {
 // otherwise, the default values of all defined flags in the set.
 func (f *FlagSet) PrintDefaults() {
 	writer := tabwriter.NewWriter(f.Out(), 20, 1, 3, ' ', 0)
+	var home string
+	if runtime.GOOS != "windows" {
```

Comment:
I actually don't want the HOME dir in general. On windows I want "home" to be "" because ~ wont' work on windows.